roles/docker/tasks/main.yml

```diff
@@ -71,6 +71,11 @@
   notify: restart docker
   when: not (ansible_os_family in ["CoreOS", "Container Linux by CoreOS"] or is_atomic) and (docker_package_info.pkgs|length > 0)
 
+- name: Enabling nvidia docker support
+  include: nvidia.yml
+  when: docker_nvidia_enabled and 'gpu-node' in group_names and ansible_distribution_release in docker_nvidia_suppored_distributions
```

Comment:
similar to the above comment. docker_nvidia_suppored_distributions -> docker_nvidia_supported_distributions

Reply:
since now (changed last month) most distro's are supported by docker nvidia no need to check on supported distro so removed this check